src/App.tsx

```diff
@@ -15,47 +18,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  let visibleGoods = [...goods];
+
+  switch (sortType) {
+    case SortType.LENGTH:
+      visibleGoods.sort((good1, good2) => {
+        return good1.length - good2.length;
+      });
+      break;
+
+    case SortType.ALPABET:
+      visibleGoods.sort((good1, good2) => {
+        return good1.localeCompare(good2);
+      });
+      break;
+
+    case SortType.NONE:
```

Comment:
```suggestion
    case SortType.NONE:
    default:
```

Reply:
not fixed
or throw an error in default case that type is not supported